### ISSUE TITLE
Support for additional hyperparameters

### ISSUE THREAD

**Issue by qychen2001** [pyg-team/pytorch-frame#416]
Very good work!
But when I wanted to use hyperparameter search for thesis result reproduction, I realized that the implementation of this repository ignores a lot of hyperparameters. For example `dropout` etc. The absence of these parameters will hinder the further use of this work in scientific comparisons.
So is it considered to add additional hyperparameters?
Below is an example of FT-Transformer showing the hyperparameter space in the original paper and the version implemented in this repository.

![image](https://github.com/pyg-team/pytorch-frame/assets/72334646/6b3b03b5-86ed-493e-8e7d-930becaaa855)
![image](https://github.com/pyg-team/pytorch-frame/assets/72334646/7cc78498-d766-4b31-bc9d-c29b1ef672fc)

For a more detailed comparison check out [the original repository implementation.](https://github.com/yandex-research/rtdl-revisiting-models/blob/main/bin/ft_transformer.py)
